src/main/java/edu/msudenver/bluejext/qualitytools/QualityToolsExtension.java

```diff
@@ -91,3 +99,42 @@ public URL getURL() {
         }
     }
 }
+
+/**
+ * Create a Preference Panel located in the BlueJ application
+ * BlueJ -> Preferences -> Extensions Tab
+ */
+class Preferences implements PreferenceGenerator
+{
+    private JPanel myPanel;
+    private JTextField color;
```

Comment:
What is the intent of this field?  It seems irrelevant.

Reply:
Field is needed to create JTextField from the Swing API.  

After discussing with the group, having a JTextField displayed in the quality tools extension tab serves no purpose.  The intent was to match the #112 user story "picture display"

new commit will remove field

Reply:
> The intent was to match the #112 user story "picture display"

Matching that with something relevant to the product, such as, "Quality Analysis Tools", would have been fine.  "color" just seems irrelevant.